### PR TITLE
RF: update some settings files

### PR DESCRIPTION
**Pull request by skoudoro**
- Remove vtk from pyproject.toml
- python 3.10 as a minimum

### CODE REVIEW

**Comment by skoudoro** [comment]
This PR is ready to go. 

A lot of follow-up PR to cleanup is needed. But this one is good as it is.

Can someone have a look @m-agour or @maharshi-gor  or @Garyfallidis ?

File: pyproject.toml [hunk 1]
```diff
@@ -33,22 +33,19 @@ classifiers = [
...
-	"scipy>=1.0",
-    "vtk>=9.1.0",
-    "pillow>=5.4.1",
-    "packaging >=17.0",
-    "pygltflib>=1.15.3",
-    "aiohttp>=3.8.4",
+    "numpy>=2.0",
+    "scipy>=1.13.0",
+    "pillow>=10.4.0",
+    "packaging >=23.0",
```
Comment (maharshi-gor) [inline pyproject.toml:48]: remove spacing from the 
```suggestion
    "packaging>=23.0",
```

Comment (maharshi-gor) [inline pyproject.toml:48]:
```suggestion
    "packaging>=23.0",
```

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -25,7 +25,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: [3.9, '3.10', 3.11, 3.12]
+        python-version: ['3.10', 3.11, 3.12, 3.13]
```
Comment (maharshi-gor) [inline .github/workflows/test.yml:28]:
```suggestion
        python-version: [3.10, 3.11, 3.12, 3.13]
```

**Review by maharshi-gor — CHANGES_REQUESTED** [review]
Hey @skoudoro  just tiny suggestions, otherwise LGTM!

File: .github/workflows/test.yml (hunk 2 above)
Comment (skoudoro) [reply .github/workflows/test.yml:28]: quote are mandatory, otherwise, it interpret it as '3.1' instead of '3.10'

**Review by maharshi-gor — APPROVED** [review]

**Pull request merged**